### PR TITLE
Add roadbike group.

### PR DESCRIPTION
**Pull request by JeldorPKU**
See #248 .

### CODE REVIEW

File: application/views/admin_lookup.php [hunk 1]
```diff
@@ -39,8 +39,8 @@
             <th>证件编号</th>
             <th>北大赛</th>
             <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
+            <th>5.6晚餐</th>
```
Comment (huxuan) [inline application/views/admin_lookup.php]: What about using `第一天晚餐` and `第二天午餐`？ [Just suggestion, not required.]

File: application/views/registration_individual.php [hunk 2]
```diff
@@ -54,18 +54,18 @@
                     <?php endforeach; ?>
                 </select>
             </div>
-            <label class="col-sm-1">5.14晚餐</label>
+            <label class="col-sm-1">5.6晚餐</label>
             <div class="col-sm-1">
                 <input type="checkbox" name="dinner">
             </div>
-            <label class="col-sm-1">5.15午餐</label>
+            <label class="col-sm-1">5.7午餐</label>
```
Comment (huxuan) [inline application/views/registration_individual.php]: same as above.

File: application/views/registration_individual.php [hunk 3]
```diff
@@ -102,8 +107,8 @@
             <th>证件编号</th>
             <th>北大赛</th>
             <th>住宿</th>
-            <th>5.14晚餐</th>
-            <th>5.15午餐</th>
+            <th>5.6晚餐</th>
+            <th>5.7午餐</th>
```
Comment (huxuan) [inline application/views/registration_individual.php]: same as above.

File: application/helpers/lib_helper.php [hunk 4]
```diff
@@ -147,3 +147,4 @@ function validate_id_number($id_number, $id_type) {
         return false;
     }
 }
+
```
Comment (huxuan) [inline application/helpers/lib_helper.php]: Remove the redundant empty line.

**Comment by huxuan** [comment]
@Luolc Please help to have another check about specific implementation.

File: application/views/admin_lookup.php (hunk 1 above)
Comment (JeldorPKU) [inline application/views/admin_lookup.php]: I think `5.6` style has no ambiguity with the date specified, which is better for comprehension. In fact we do not have races on the first day.

Comment (huxuan) [inline application/views/admin_lookup.php]: OK, it convince me well.

File: application/models/people_model.php [hunk 5]
```diff
@@ -153,6 +153,14 @@ public function get_all_people() {
         }
     }
 
+    /*
+     * Get remained quota of RDB
+     */
+    public function get_rdb_quota() {
+        $query = $this->db->where('deleted', false)->where('rdb', true)->get('people');
+        return 80 - $query->num_rows();
```
Comment (Luolc) [inline application/models/people_model.php]: Instead of hard-coding, it is better to put the constant `80` in the config file.

Comment (JeldorPKU) [inline application/models/people_model.php]: Done.

**Pull request merged**